### PR TITLE
Bug 1980182: operator UI not working with multiple version CRD in 4.6

### PR DESCRIPTION
**Pull request by florkbr**
Cherry picks pull request #7305 ([Bug 1888036](https://bugzilla.redhat.com/show_bug.cgi?id=1888036)) from @TheRealJon to the `release-4.6` branch to resolve this issue. The cherry pick was not clean (some files did not exist back in 4.6), hence the difference in lines in the diff.

Tested locally using a 4.6 cluster and seems to do the trick. I modified an existing operators yaml and CRD to reproduce. See screenshots below.

https://bugzilla.redhat.com/show_bug.cgi?id=1980182

## Setting up the 4.6 cluster
Screen Shot 2021-07-19 at 4 37 11 PM
Screen Shot 2021-07-19 at 4 39 16 PM

## Reproduced issue
Screen Shot 2021-07-19 at 4 34 18 PM

## With fix
Screen Shot 2021-07-19 at 4 40 52 PM

### CODE REVIEW

**Review by TheRealJon — APPROVED** [review]
/lgtm
/approve

**Comment by florkbr** [comment]
@TheRealJon @spadgett tide is saying we're missing labels... do we need to add those manually?

image

**Comment by TheRealJon** [comment]
@florkbr No, the `cherry-pick-approved` label will be added by the appropriate approver, and the `approved` label will be added when one other owner approves this PR.

**Review by spadgett — APPROVED** [review]
/approve

**Pull request merged**
